src/Misc/layoutbin/powershell/Add-WindowsSdkCapabilities.ps1

```diff
@@ -7,12 +7,19 @@ $windowsSdks = @( )
 $windowsSdkKeyName = 'Software\Microsoft\Microsoft SDKs\Windows'
 $versionSubKeyNames =
     Get-RegistrySubKeyNames -Hive 'LocalMachine' -View 'Registry32' -KeyName $windowsSdkKeyName |
-    Where-Object { $_ -clike 'v*A' }
+    Where-Object { $_ -clike 'v*A' -or $_ -clike 'v10*' }
 foreach ($versionSubKeyName in $versionSubKeyNames) {
     # Parse the version.
     $version = $null
-    if (!([System.Version]::TryParse($versionSubKeyName.Substring(1, $versionSubKeyName.Length - 2), [ref]$version))) {
-        continue
+    if (!($versionSubKeyName -clike '*A')) {
```

Comment:
Would this be more readable if we didn't negate this check and swapped the if and the else?

Reply:
Good call - done